src/component/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+import classNames from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  status: Status;
+  setStatus: (status: Status) => void;
+};
+
+export const Footer: React.FC<Props> = ({ todos, status, setStatus }) => {
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+  const hasCompleted = todos.some(todo => todo.completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      {todosLeft === 1 ? (
+        <span className="todo-count" data-cy="TodosCounter">
+          {`${todosLeft} item left`}
+        </span>
+      ) : (
+        <span className="todo-count" data-cy="TodosCounter">
+          {`${todosLeft} items left`}
+        </span>
+      )}
```

Comment:
diff only in letter 's'

```suggestion
       (
        <span className="todo-count" data-cy="TodosCounter">
          {`${todosLeft} item${todosLeft === 1 ? '' : 's'} left`}
        </span>
      )
```